Forta-Bots-Challenge-1/src/constants.ts

```diff
@@ -0,0 +1,7 @@
+export const CREATE_AGENT_SIGNATURE =
```

Comment:
Technically, these aren't function signatures. Just a note as it could be important so as to not confuse someone's expectations are they're reading through the code.